src/main/java/com/cloudbees/jenkins/GitHubRepositoryNameContributor.java

```diff
@@ -55,8 +55,8 @@ protected EnvVars buildEnv(AbstractProject<?, ?> job) {
             for (EnvironmentContributor contributor : EnvironmentContributor.all()) {
                 try {
                     contributor.buildEnvironmentFor(job, env, TaskListener.NULL);
-                } catch (Exception e) {
-                    LOGGER.debug("{} failed to build environment ({})", contributor.getClass(), e.getMessage());
+                } catch (Throwable t) {
```

Comment:
After the discussions in other PRs we agreed that this approach is not good, because it suppresses various kinds of system <code>Error</code>s, which are supposed to be system failures